modules/apps/portal-scheduler/portal-scheduler/src/main/java/com/liferay/portal/scheduler/internal/SchedulerEngineHelperImpl.java

```diff
@@ -174,9 +175,15 @@ public String getCronText(
 		Calendar recurrenceCalendar = null;
 
 		if (timeZoneSensitive) {
-			recurrenceCalendar = CalendarFactoryUtil.getCalendar();
+			String timeZoneId = ParamUtil.getString(
+				portletRequest, "timeZoneId");
 
-			recurrenceCalendar.setTime(calendar.getTime());
+			if (Validator.isNotNull(timeZoneId)) {
```

Comment:
Hi @Alec-Shay,

I do not think you need to set the timezone here, we only want to generate the corn text which is later used in trigger, the corn text does not need to care timezone at all. 

Also, if you go into the quartz code, you will see the timezone is going to reset anyway.

 Could you please revert this change and test if it works?

Thanks.
Tina.

Reply:
@tinatian Sorry for the delay in responding. I have not had a chance to test this yet since I am working on an urgent ticket, but I hope to get to this soon.

Reply:
@tinatian I've looked at this and tested again, and you're right, it doesn't seem to be a necessary change. I thought it was at the time of my original testing but didn't realize it ends up getting reset later on.

Would you want me to send a new PR with this reverted?

Reply:
@Alec-Shay  Yes, please!

Thanks.
Tina

Reply:
New PR sent here: https://github.com/tinatian/liferay-portal/pull/1428